soci/fbs/ztoc.fbs

```diff
@@ -0,0 +1,47 @@
+namespace ztoc;
+
+table Xattr {
+	key : string;
+	value : string;
+}
+
+table FileMetadata {
+	name : string;
+	type : string;
+	uncompressedOffset : long;
+	uncompressedSize : long;
+	spanStart : int;
+	spanEnd : int;
+	firstSpanHasBits : bool;    // Flag for if there is partial uncompressed data that is stored in the previous byte
+	linkname : string;          // Target name of link (valid for TypeLink or TypeSymlink)
+	mode : long;                // Permission and mode bits
+	uid : int;                  // User ID of owner
+	gid : int;                  // Group ID of owner
+	uname : string;             // User name of owner
+	gname : string;             // Group name of owner
+
+	modTime  : string;          // Modification time
```

Comment:
We should include the expected datetime format here.

Reply:
Done.